lib/commands/context.js

```diff
@@ -396,7 +396,8 @@ async function openNewPage () {
 }
 
 extensions.typeAndNavToUrl = async function () {
-  this.setCurrentUrl(this.caps.safariInitialUrl || `http://127.0.0.1:${this.opts.port}/welcome`);
+  let address = (this.opts.address === '0.0.0.0') ? '127.0.0.1' : this.opts.address;
```

Comment:
Does this not work just keeping `0.0.0.0` for the base case? It seems to on my machine.